src/providers/api.service.js

```diff
@@ -123,6 +125,66 @@ function HereMapsAPIService($q, $http, HereMapsConfig, HereMapsUtilsService, Her
         return deferred.promise;
     }
 
+    function geocodeAutocomplete(params) {
+        if (!params)
+            return console.error('Missing required parameters');
+
+        var autocompleteUrl = protocol + '://autocomplete.geocoder.cit.api.here.com/6.2/suggest.json',
```

Comment:
@aanev please move an url to CONFIG constant above of this file here: https://github.com/dverbovyi/angular-heremaps/blob/master/src/providers/api.service.js#L19